Source/Scene/ModelInstanceCollection.js

```diff
@@ -580,6 +586,7 @@ define([
                 modelOptions.cacheKey = collection._url + '#instanced';
             }
         } else {
+            // TODO : does non-instancing path need to set the cache key?
```

Comment:
Do you want to address this now or later?  Either is OK with me until we merge into `master`.